src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/processors/hmcts/CourtCaseEventsProcessor.kt

```diff
@@ -95,20 +93,9 @@ class CourtCaseEventsProcessor(
       ),
     )
 
-    val pageablePersonEntities: Page<PersonEntity> = personService.findCandidateRecords(person)
-    telemetryService.trackEvent(
-      CPR_CANDIDATE_RECORD_SEARCH,
-      mapOf(
-        EventKeys.SOURCE_SYSTEM to SourceSystemType.LIBRA.name,
-        EventKeys.RECORD_COUNT to pageablePersonEntities.totalElements.toString(),
-        EventKeys.EVENT_TYPE to LIBRA_COURT_CASE.name,
-        EventKeys.MESSAGE_ID to sqsMessage.messageId,
-        EventKeys.SEARCH_VERSION to PersonSpecification.SEARCH_VERSION,
-      ),
-    )
     personService.processMessage(person) {
```

Comment:
This looks odd - we have 3 (sort of) consecutive calls to different methods on PersonService.